app/helpers/automated_tests_helper.rb

```diff
@@ -1,6 +1,6 @@
 module AutomatedTestsHelper
   def extra_test_group_schema(assignment)
-    criterion_names, criterion_disambig = assignment.get_criteria(:ta).map do |c|
+    criterion_names, criterion_disambig = assignment.ta_criteria.map do |c|
       [c.name, "#{c.id}_#{c.class.name}"]
```

Comment:
This data can be simplified, make the second element just the `c.id` (and not a string).